launch_alerts.py

```diff
@@ -159,6 +159,14 @@ async def send_launch_alert(lm: LaunchMonitor) -> None:
 
     asyncio.ensure_future(send_launch_panel(channel, launch, config.timezone, message="There's a launch coming up!"))
 
+    async for msg in channel.history(limit = 3):
+        if msg.embeds and msg.author == bot.user:
+            if msg.embeds[0].title == launch["name"]:
```

Comment:
The embed has the slug on it which should never change for a specific launch.  I think it would be better to parse that out instead of comparing titles.

Reply:
Okay, let me work on getting the slug out. That sounds better than the title for sure.

Reply:
Updated